pkg/chain/local/local.go

```diff
@@ -306,6 +306,7 @@ func ConnectWithKey(
 			TicketSubmissionTimeout:    6,
 			ResultPublicationBlockStep: 3,
 			MinimumStake:               minimumStake,
+			RelayEntryTimeout:          12,
```

Comment:
Why 12? It depends on `groupSize`, right? Where do we need it?

Reply:
Ah right, I changed.